styles/typography.ts

```diff
@@ -65,6 +65,10 @@ export const SubTitle = styled.h3<TextType>`
     props.styling ? props.styling : "normal"};
   align-self: ${(props: TextType) => (props.self ? props.self : "flex-start")};
   margin: ${(props: TextType) => (props.margins ? props.margins : undefined)};
+  ${(props) => 
+    props.mobMargins 
+      ? breakpoints("margin", "", [{ 1000: props.mobMargins }])
+      : breakpoints("margin", "", [{ 1000: "1rem 1rem 0 1rem" }])}
```

Comment:
the last part of the tertiary as in the default breakpoint if no mobMargins is passed is kind of super specific for a default style when no style is passed. do we want all subtitles everywhere to have 1rem everywhere but on the bottom at 1000px screenwidth

Reply:
not 100% sure, probably good to discuss